pinot-hadoop/src/main/java/org/apache/pinot/hadoop/utils/PushLocation.java

```diff
@@ -18,13 +18,25 @@
  */
 package org.apache.pinot.hadoop.utils;
 
+import java.util.ArrayList;
+import java.util.List;
+
+
 public class PushLocation {
   private final String _host;
   private final int _port;
 
-  private PushLocation(PushLocationBuilder pushLocationBuilder) {
-    _host = pushLocationBuilder._host;
-    _port = pushLocationBuilder._port;
+  public PushLocation(String host, int port) {
```

Comment:
This change may break hadoop jobs for people out side if they do something similar to PBNJ (importing pinot-hadoop jar for their hadoop job project). For now, I think that this is fine; however, it's going to be hard for us to change the constructor in very near future. Once we publish our jar files to public maven repo, this kind of change should happen in multiple steps. (Add constructor, add deprecated annotation, remove them with the major version bump).

Reply:
Good point. Once we go with official release, these kind of changes need to be out in multiple steps.